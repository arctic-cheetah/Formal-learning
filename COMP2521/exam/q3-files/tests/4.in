from: 7 
2 7
1 6
4 2 
2 5 
3 7
5 1
1 7 


 

