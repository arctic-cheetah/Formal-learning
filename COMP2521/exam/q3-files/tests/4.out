rankPopularity, From: 7 

Edge inserted:  2 -> 7 
Edge inserted:  1 -> 6 
Edge inserted:  4 -> 2 
Edge inserted:  2 -> 5 
Edge inserted:  3 -> 7 
Edge inserted:  5 -> 1 
Edge inserted:  1 -> 7 


rankPopularity: 
 node: 7, rank:      6.000 
 
